documentation/b2b.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: master
+permalink: /documentation/b2b/
+title: B2B
+weight: 7
+---
+# B2B Storefront Configuration and Implementation
+
+## Configuration
+TBA
+
+## KeyCloak Theme Implementation
+By default, KeyCloak provides a set of themes that may be used for the various actions KeyCloak supports (login, accounts, etc). Either of these existing themes may be used for the supported flows, or a customized theme may be added to the KeyCloak Docker Image if desired. The REACT PWA Reference Storefront includes a login theme that may be used for this purpose.
+
+The current KeyClock Docker Image will expect any cusom themes to be provided in the directory `/devops/docker/keycloak/themes`.
```

Comment:
For the current KeyClock Docker Image, add the custom themes to the `/devops/docker/keycloak/themes` directory .